packages/fuels-contract/src/contract_calls_utils.rs

```diff
@@ -256,19 +256,22 @@ fn convert_to_signed_resources(spendable_resources: Vec<Resource>) -> Vec<Input>
         .collect()
 }
 
+/// Get the base offset for a script
+pub fn get_base_script_offset() -> usize {
+    ConsensusParameters::DEFAULT.tx_offset() + fuel_tx::Script::script_offset_static()
```

Comment:
So I think we should rather query the consensus parameters here rather than using a default.

Reply:
This is just for the offset. We do not use the real values.

Reply:
Oh you mean the `tx_offset` value does not change with the consensus parameters? If so I think it's useful to clarify it.

Reply:
You are right it does change. But it only depends on `max_inputs` that is why we did not have problems until now :/

Reply:
This was a great catch. If the `max_inputs` parameter is changes our contract calls break as the offset is not ok. I will refactor the code a bit to take the consensus_parameter from the node into account.